control-plane/grpc/src/operations/registry/traits.rs

```diff
@@ -102,6 +107,14 @@ impl TryFrom<registry::Specs> for transport::Specs {
                 }
                 replica_specs
             },
+            volume_groups: {
+                let mut vg_specs: Vec<VolumeGroupSpec> =
+                    Vec::with_capacity(value.volume_groups.len());
+                for vg_spec in value.volume_groups {
+                    vg_specs.push(VolumeGroupSpec::try_from(vg_spec)?);
+                }
+                vg_specs
```

Comment:
```suggestion
value.volume_groups.into_iter().map(VolumeGroupSpec::try_from).collect::<Result<Vec<VolumeGroupSpec>, SvcError>>()?
```

Reply:
Good point. Changed it for the other resources as well.